packages/react/src/components/Tabulation/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+class Tabulation extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeTabIndex: null,
```

Comment:
What about setting `0` as default index so the first element will be selected by default?

Reply:
I wouldn't do that as I think it's not a good idea, to impose which tab is opened by default. Maybe there's a case, where nothing would be opened?

But! Your track is good. How about introducing additional prop, which would override that `null` if there's a need to?